### ISSUE TITLE
Debug vote service in backend

### ISSUE THREAD

**Issue by jsangmeister** [OpenSlides/openslides-vote-service#17]
In https://github.com/OpenSlides/openslides-backend/pull/677 I included the vote service in the backend, which worked fine until #1 was merged (I think). Now, whenever the vote service sends a request to the Postgres database, it fails with a `"closed pool"` error. Steps to reproduce:
1. Checkout https://github.com/jsangmeister/openslides-backend/tree/vote-service
2. `make run-dev`
3. Create the poll by executing `pytest -k test_setup_models` (I added a dummy test for easier creation)
4. Make a request to the vote service with `curl -X POST vote:9013/internal/vote/create?id=1`
You can check the logs of the vote service by executing `docker container logs dev_vote_1` on the host (or `dev_postgresql_1` for postgres). It seems like postgres is never even reached, at least no logs appear when executing the request.

I am not sure whether the issue is docker (setup) related, but since the database is reached initially (if you give e.g. another host name, the vote service does not even start) I assume it is a vote service error. @ostcar (can't assign you here)

**Comment by ostcar** [comment]
@jsangmeister #1 introduced the postgres backend. Before this change, the redis backend was used, even for `long` polls. This means, before #1 a postgres setup was not necessary.

Currently, the postgres backend is used as default: https://github.com/OpenSlides/OpenSlides/blob/4fc44929f87c114a14f1451eb4a146020588a6eb/docs/models.yml#L2509

To change this, you can set the backend-attribute of the poll in the datastore to "fast".

A better way would be to fix the postgres setup. Maybe the environment varialbes are wrong?  https://github.com/OpenSlides/openslides-vote-service/blob/9fd738a85bcc555bf2eb4d109e71594801fcdb19/README.md?plain=1#L132-L134

**Comment by jsangmeister** [comment]
Yes, thanks, with the `fast` backend it works. I will check the variables.
